src/srr-mapping.ts

```diff
@@ -67,7 +67,9 @@ export function handleTransfer(event: TransferEvent): void {
   srr.originChain = currentChainId();
   srr.originTxHash = event.transaction.hash;
 
-  srr.createdAt = timestampMillis;
```

Comment:
Good these changes should be in develop. Wondering though if any recent release to QA from develop is missing these? Could you check. In any case this PR should be merged back to develop.